contracts/MemeFactory.sol

```diff
@@ -316,14 +309,11 @@ abstract contract MemeFactory {
         // Check the unleash timestamp
         require(block.timestamp >= memeSummon.summonTime + UNLEASH_DELAY, "Cannot unleash yet");
 
-        // Wrap native token to its ERC-20 version, where applicable
-        _wrap(totalNativeTokenCommitted);
-
         // Put aside reference token to buy OLAS with the burn percentage of the total native token amount committed
         uint256 nativeAmountForOLASBurn = (totalNativeTokenCommitted * OLAS_BURN_PERCENTAGE) / 100;
 
         // Adjust reference token amount
-        totalNativeTokenCommitted -= nativeAmountForOLASBurn;
+        uint256 nativeAmountForLP = totalNativeTokenCommitted - nativeAmountForOLASBurn;
```

Comment:
There was a bug here. Relabelling makes this apparent.

Reply:
Fine, but at the end it's the same math.

Reply:
Ah I see now.

Reply:
No the amount passed to _collect ended up being different depending on whether called from unleashed or collect